uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/OrchestratorZoneControllerIntegrationTests.java

```diff
@@ -481,6 +483,94 @@ public void testCreateZone_ZoneAlreadyExists() {
+        expectedResponse.setName(zoneId);
+        expectedResponse.setMessage(ZONE_CREATED_MESSAGE);
+        expectedResponse.setState(OrchestratorState.CREATE_IN_PROGRESS.toString());
+
+        assertEquals(HttpStatus.ACCEPTED, response.getStatusCode());
+        assertResponse(expectedResponse, response.getBody());
+    }
+
+    @Test
+    public void testImportZone_IdentityZoneNotPresent() {
+        String zoneId = UUID.randomUUID().toString();
+        OrchestratorZoneRequest orchestratorZoneRequest = getOrchestratorImportZoneRequest(zoneId, ADMIN_CLIENT_SECRET,
+                ZONE_SUBDOMAIN, zoneId);
+        ResponseEntity<OrchestratorZoneResponse> getIdentityZoneNotPresentResponse = client.exchange(
+                serverRunning.getUrl(ORCHESTRATOR_ZONES_APIS_ENDPOINT),
+                HttpMethod.POST,
+                new HttpEntity<>(orchestratorZoneRequest),
+                OrchestratorZoneResponse.class);
+
+        OrchestratorZoneResponse expectedResponse = new OrchestratorZoneResponse();
+        expectedResponse.setName(zoneId);
+        expectedResponse.setState(OrchestratorState.PERMANENT_FAILURE.toString());
+
+        assertEquals(HttpStatus.NOT_FOUND, getIdentityZoneNotPresentResponse.getStatusCode());
+        assertTrue(getIdentityZoneNotPresentResponse.getBody().getMessage().contains("Zone[" + zoneId + "] not found."));
+    }
+
+    @Test
+    public void testImportZone_IdentityZoneAlreadyPresentInOrchestratorZone() {
+        String zoneId = UUID.randomUUID().toString();
+        ResponseEntity<Void> createIdentityZoneResponse = createIdentityZone(zoneId);
+        Assert.assertEquals(HttpStatus.CREATED, createIdentityZoneResponse.getStatusCode());
+
+        ResponseEntity<OrchestratorZoneResponse> firstImportResponse = importZone(zoneId, zoneId);
+
+        OrchestratorZoneResponse expectedFirstImportResponse = new OrchestratorZoneResponse();
+        expectedFirstImportResponse.setName(zoneId);
+        expectedFirstImportResponse.setMessage(ZONE_CREATED_MESSAGE);
+        expectedFirstImportResponse.setState(OrchestratorState.CREATE_IN_PROGRESS.toString());
+
+        assertEquals(HttpStatus.ACCEPTED, firstImportResponse.getStatusCode());
+        assertResponse(expectedFirstImportResponse, firstImportResponse.getBody());
+
+        String newZoneName = UUID.randomUUID().toString();
+        ResponseEntity<OrchestratorZoneResponse> secondImportresponse = importZone(newZoneName, zoneId);
+
+        OrchestratorZoneResponse expectedSecondImportResponse = new OrchestratorZoneResponse();
+        expectedSecondImportResponse.setName(zoneId);
+
+        assertEquals(HttpStatus.CONFLICT, secondImportresponse.getStatusCode());
+        assertTrue(secondImportresponse.getBody().getMessage().contains("already present in orchestrator zone, Import not needed"));
+    }
+
+    @Test
+    public void testImportZone_ZoneNameAlreadyExist() {
```

Comment:
Both testImportZone_IdentityZoneAlreadyPresentInOrchestratorZone and testImportZone_ZoneNameAlreadyExist are duplicate test case. Remove any one.